### DISCUSSION TITLE
designable组件的size属性在多语言显示中固定了选项顺序

### DISCUSSION THREAD

**Discussion by aloha66** [alibaba/designable#29] [General]
不知道应该在这边还是在formily建一个issue。
[复现链接](https://designable-antd.formilyjs.org/)
## 重现步骤:
1. 选择switch组件，拖动到workspace
2. 在右侧属性配置的尺寸调整大小
3. 观察workspace的switch变化

## 实际的结果是什么

操作结果与代码一致（enum: ['small', 'default'],），只是显示的文案有错

useLocales.js
![image](https://user-images.githubusercontent.com/20942781/127251357-22db735a-5fef-408b-b21f-0a7fc559c92e.png)
在尺寸的field.dataSource，固定了显示顺序

**Comment by janryWang** [comment]
嗯，是个Bug，已经修复了，后面直接提到formily discussion里吧

**Reply by aloha66** [reply]
迅速啊
